src/VisualStudio/LiveShare/Impl/LiveShareConstants.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.LanguageServices.LiveShare
+{
+    class LiveShareConstants
+    {
+        // The service name for an LSP server implemented using Roslyn designed to be used with the Roslyn client
+        public const string RoslynContractName = "Roslyn";
+        // The service name for an LSP server implemented using Roslyn designed to be used with the LSP SDK client
+        public const string RoslynLSPSDKContractName = "RoslynLSPSDK";
```

Comment:
what is RoslynLSPSDK?

Reply:
It's a different contract name that they use to export part of the diagnostic handler.  I'm not sure exactly what they use it for.  I'll find out